app/scenes/root/root.js

```diff
@@ -41,7 +38,8 @@ export default class Root extends PureComponent {
     componentWillReceiveProps(nextProps) {
         if (this.props.logoutRequest.status === RequestStatus.STARTED &&
             nextProps.logoutRequest.status === RequestStatus.SUCCESS) {
-            this.props.actions.removeStorage();
+
+            // TODO: Purse storage here
```

Comment:
Haha definitely purge. We'll handle keeping the serverUrl a different way. I'll make change.